lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -142,6 +142,8 @@
 INSTRUCTOR_GET_ENDPOINTS = {
     'get_anon_ids',
     'get_issued_certificates',
+    'instructor_api_v0:list_instructor_tasks',
+    'instructor_api_v0:list_report_downloads',
```

Comment:
Originally, these were POST endpoints, however they are GET endpoints under the new API since they simply list items, and potentially a future API could add support for other operations.